extension/src/NABfunctions.ts

```diff
@@ -196,24 +196,24 @@ export async function findNextUnTranslatedText(
       );
     }
     // Run refresh from g.xlf then run again.
-    if (!foundAnything) {
-      await refreshXlfFilesFromGXlf();
-      foundAnything = await LanguageFunctions.findNextUnTranslatedText(
-        settings,
-        SettingsLoader.getAppManifest(),
-        false,
-        languageFunctionsSettings.replaceSelfClosingXlfTags,
-        lowerThanTargetState
-      );
+    if (languageFunctionsSettings.refreshXlfAfterFindNextUntranslated) {
+      if (!foundAnything) {
+        await refreshXlfFilesFromGXlf();
```

Comment:
Hmmm... Just realized that this line will always show an information message, even if nothing was changed. Should we suppress the message from refreshXlfFilesFromGXlf() if nothing was updated? Because in that case we would show the "No more untranslated texts found" message below...

Reply:
If this is too challenging to do now, we could do it later if it's getting too annoying...?

Reply:
nearly done ;)

Reply:
Found a bug :D

Reply:
So, Refresh XLF would show a message stating "Nothing changed in x XLF files" even tho review tokes were added for for missing placeholders, optionstrings etc. This is now mitigated, it may not be the ultimate solution but it's not very intrusive so it shouldn't be a big deal reworking it :)